docs/docs/30-administration/15-agent-config.md

```diff
@@ -80,6 +80,11 @@ Disable colored debug output.
 
 Configures the agent hostname.
 
+### `WOODPECKER_AGENT_ID_FILE`
```

Comment:
we should add to all examples the volume `/etc/woodpecker/` so it is persistent

Reply:
I added it as an example for the other file config of the agent: `WOODPECKER_AGENT_SECRET_FILE`